### ISSUE TITLE
add Confirmation Dialog Box for "replace changes" Button

### ISSUE THREAD

**Issue by creeloper27** [syncthing/syncthing-android#1447]
Story:
I downloaded and synced a directory from my computer to my phone, it wasn't syncthing and I didn't realise the problem with the external sd card on android, I saw the "replace changes" button appear and seeing that it was not syncthing i pressed it, I immidiatly got all the files deleted because the folder on the sd card was empty, fortunatly i use file versioning on my computer and i was able to restore.

Bottom Line:
I think there must be a confimation box for the "replace changes" button, espetially if the folder is empty, the user must be notified that all his files will be deleted from the other devices where the folder is shared as a resoult of that action.
